hail/python/test/hail/helpers.py

```diff
@@ -121,6 +121,16 @@ def create_all_values_datasets():
     return (create_all_values_table(), create_all_values_matrix_table())
 
 
+def run_in(*locations, clouds=None):
+    for location in locations:
+        if location not in ('local', 'spark', 'batch', 'all'):
+            raise RuntimeError(f'unknown location {location}')
```

Comment:
let's error if 'all' is specified with any other location.

Reply:
Maybe also error if all three are specified? Best to enforce a consistent, simple style across everything.